app/helpers.php

```diff
@@ -116,6 +116,38 @@ function es_records($results, $class)
     return $records;
 }
 
+function es_search($params)
+{
+    try {
+        return Es::search(array_merge_recursive([
+            'client' => [
+                'timeout' => 10,
+                'connect_timeout' => 5,
```

Comment:
are these seconds? if so, these are probably way too high

Reply:
maybe `connect_timeout => 0.5`? I don't think we want `timeout` itself to be too low since that's the timeout for the entire operation whether it's completed or not.

Reply:
If Elasticsearch is taking _seconds_ to return results, there's probably something seriously wrong...?

Looking at our search metrics, nothing has even come close to even 1s (except user search which isn't Elasticsearch iirc) so maybe 1-2s is fine for timeout?

Reply:
and then it lags in dev, so this should probably be a config setting :v